shiftregister/shiftregister.go

```diff
@@ -0,0 +1,66 @@
+// Package shiftregister is for 8bit shift output register using 3 GPIO pins
+//
+// Tested with SN74HC595 with 8 outputs
```

Comment:
Looks like the driver is compatible with several ICs, better to avoid specific wiring or datasheet, just name some of the compatible ICs like : https://github.com/tinygo-org/drivers/blob/master/shifter/shifter.go#L1

Reply:
I just have this one SN74HC595. No idea about  possible other references 

>     * Modify the comment/header of the driver file, to make it more general, use [this](https://github.com/tinygo-org/drivers/blob/master/shifter/shifter.go#L1) as an example

Reply:
I'm not sure either, but here are some of the most common I found: SN74ALS164A , SN74AHC594 , SN74AHC595, ...